docker-compose.yml

```diff
@@ -176,6 +175,7 @@ services:
     volumes:
       - ./msar/static:/code/static
       - ./msar/media:/code/media
+      - ./msar/secrets:/code/.secrets
```

Comment:
Please add documentation about the `secrets` volume in the "INFO ABOUT VOLUMES" section.

Reply:
Good catch.